library/Booster/CLOptions.hs

```diff
@@ -2,12 +2,14 @@
 
 module Booster.CLOptions (
     CLOptions (..),
+    EquationOptions (..),
     clOptionsParser,
     adjustLogLevels,
     versionInfoParser,
 ) where
 
 import Booster.Trace (CustomUserEventType)
+import Booster.Util (Bound (..))
 import Booster.VersionInfo (VersionInfo (..), versionInfo)
```

Comment:
:thinking: maybe not too important but I usually put the library imports first, and our "own" imports afterwards (separated by a blank line). These imports were obviously not like that before so nvm... :shrug:

Reply:
yeah, I've noticed, but I though it does not matter that much to drag another change into this PR